spec.html

```diff
@@ -5069,9 +5069,9 @@ <h1>SameValueNonNumeric ( _x_, _y_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-abstract-relational-comparison" aoid="Abstract Relational Comparison">
-      <h1>Abstract Relational Comparison</h1>
-      <p>The comparison _x_ &lt; _y_, where _x_ and _y_ are values, produces *true*, *false*, or *undefined* (which indicates that at least one operand is *NaN*). In addition to _x_ and _y_ the algorithm takes a Boolean flag named _LeftFirst_ as a parameter. The flag is used to control the order in which operations with potentially visible side-effects are performed upon _x_ and _y_. It is necessary because ECMAScript specifies left to right evaluation of expressions. The default value of _LeftFirst_ is *true* and indicates that the _x_ parameter corresponds to an expression that occurs to the left of the _y_ parameter's corresponding expression. If _LeftFirst_ is *false*, the reverse is the case and operations must be performed upon _y_ before _x_. Such a comparison is performed as follows:</p>
+    <emu-clause id="sec-abstract-relational-comparison" aoid="AbstractRelationalComparison">
+      <h1>AbstractRelationalComparison ( _x_, _y_ [ , _LeftFirst_ ] )</h1>
+      <p>The abstract operation AbstractRelationalComparison takes arguments _x_ (an ECMAScript language value) and _y_ (an ECMAScript language value), and optional argument _LeftFirst_ (a Boolean). It provides the semantics for the comparison _x_ &lt; _y_, returning *true*, *false*, or *undefined* (which indicates that at least one operand is *NaN*). The _LeftFirst_ flag is used to control the order in which operations with potentially visible side-effects are performed upon _x_ and _y_. It is necessary because ECMAScript specifies left to right evaluation of expressions. The default value of _LeftFirst_ is *true* and indicates that the _x_ parameter corresponds to an expression that occurs to the left of the _y_ parameter's corresponding expression. If _LeftFirst_ is *false*, the reverse is the case and operations must be performed upon _y_ before _x_. It performs the following steps when called:</p>
       <emu-alg>
         1. If the _LeftFirst_ flag is *true*, then
```

Comment:
I didn't realize we ever defaulted parameters by prose in the preamble. That seems not great (although unrelated to this change).

Reply:
I'm guessing you mean: state a default in the preamble *without* 'implementing' that default via a presence-check in the algorithm. If so: yes, not great.

Reply:
We do that in a number of places, yes (see #2379, for example)